frontend/src/components/common/LoginBtn/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,231 @@
+                    </p>
+                  )}
+                  {submitted && email && !validateEmail(email) && (
+                    <p className={styles.errorMessage}>
+                      Невірний формат електронної пошти
+                    </p>
+                  )}
+                  {resetPassword && !email && (
+                    <p className={styles.errorMessage}>
+                      Введіть електронну пошту
+                    </p>
+                  )}
+                  {resetPassword && email && !validateEmail(email) && (
+                    <p className={styles.errorMessage}>
+                      Невірний формат електронної пошти
+                    </p>
+                  )}
+                  {submitted && resetEmailError && (
+                    <p className={styles.errorMessage}>{resetEmailError}</p>
+                  )}
+                </div>
+
+                <div className={styles.passwordContainer}>
+                  <label className={styles.labelData} htmlFor="password">
+                    Пароль {submitted && !password && <span>*</span>}
+                  </label>
+
+                  <input
+                    id="password"
+                    name="password"
+                    className={styles.dataInput}
+                    type="password"
+                    placeholder="Введіть ваш пароль"
+                    value={password}
+                    onChange={(e) => setPassword(e.target.value)}
+                  />
+                  {submitted && !password && (
+                    <p className={styles.errorMessage}>Введіть пароль</p>
+                  )}
+                  {submitted && passwordError && (
+                    <p className={styles.errorMessage}>{passwordError}</p>
+                  )}
+                </div>
+              </div>
+
+              <div className={styles.button}>
+                <Button
+                  variant="primary"
+                  label="Увійти"
+                  padding="padding-sm"
+                  isFullWidth={true}
+                  type="submit"
+                  onClick={handleSubmit}
+                />
+              </div>
+              <div>
+                <p>або</p>
+              </div>
+              <div className={styles.googleLogin}>
+                <GoogleLogin
+                  handleGoogleLogin={(token) => {
+                    handleGoogleLogin(token);
+                    toggleLogin();
+                  }}
+                />
+              </div>
+
+              <div className={styles.resetPassword}>
+                <button
+                  className={styles.resetPassword}
+                  onClick={handleResetPassword}
+                  type="submit"
+                >
+                  Забули пароль?
+                </button>
+              </div>
+              {resetPasswordClicked &&
+                resetPassword &&
+                email &&
+                validateEmail(email) && (
```

Comment:
Better to move it to a variable for readability

Reply:
done